ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -614,7 +639,7 @@ public static CCDStatementOfMeans getCCDStatementOfMeans() {
     }
 
     public static CCDCase getCCDCitizenCaseWithOperationIndicators(
-                 Supplier<CCDClaimSubmissionOperationIndicators> claimIndicatorSupplier) {
+        Supplier<CCDClaimSubmissionOperationIndicators> claimIndicatorSupplier) {
```

Comment:
I know it's not your code, but Supplier here in parameters looks unnecessary, not used as the assignment target for a lambda expression or method reference any where. Parameter could have passed simply of type `CCDClaimSubmissionOperationIndicators`.